documentation/asciidoc/computers/camera/troubleshooting.adoc

```diff
@@ -0,0 +1,16 @@
+== Troubleshooting
+
+If your Camera Module doesn't work like you expect, try some of the following fixes:
+
+* On Raspberry Pi 3 and earlier devices running Raspberry Pi OS _Bullseye_ or earlier:
+** To enable hardware-accelerated camera previews, enable *Glamor*. To enable Glamor, enter `sudo raspi-config` in a terminal, select `Advanced Options` > `Glamor` > `Yes`. Then reboot your Raspberry Pi with `sudo reboot`.
+** If you see an error related to the display driver, add `dtoverlay=vc4-fkms-v3d` or `dtoverlay=vc4-kms-v3d` to `/boot/config.txt`. Then reboot your Raspberry Pi with `sudo reboot`.
+** The graphics hardware can only support images up to 2048×2048 pixels, which places a limit on the camera images that can be resized into the preview window. As a result, video encoding of images larger than 2048 pixels across produces corrupted or missing preview images.
```

Comment:
In this line and the next, perhaps "pixels wide" is better terminology than "pixels across" ?